Routing/RestRouteCollection.php

```diff
@@ -76,10 +76,17 @@ public function setDefaultFormat($format)
      */
     public function all()
     {
+        $regex = <<<REGEX
```

Comment:
call me crazy, but this looks IMHO odd, too :D 
But hey, because of that the following rule exists: http://www.php-fig.org/psr/psr-2/ (120 char limit)